utils/network.go

```diff
@@ -29,8 +29,39 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
+var (
+	// IPv4AllNet represents the IPv4 all-addresses CIDR 0.0.0.0/0.
+	IPv4AllNet *net.IPNet
+	// IPv6AllNet represents the IPv6 all-addresses CIDR ::/0.
+	IPv6AllNet    *net.IPNet
+	DefaultRoutes []*net.IPNet
+)
+
+func init() {
+	var err error
+	_, IPv4AllNet, err = net.ParseCIDR("0.0.0.0/0")
+	if err != nil {
+		panic(err)
```

Comment:
The app will crash with a panic.

Reply:
I guess I was worrying if that would be a new scenario; if we know how k8s as a whole reacts if that happens...

But never mind, because this is a panic that really *won't* ever happen anyway.